mmdet/models/detectors/retinanet.py

```diff
@@ -15,3 +16,21 @@ def __init__(self,
                  pretrained=None):
         super(RetinaNet, self).__init__(backbone, neck, bbox_head, train_cfg,
                                         test_cfg, pretrained)
+
+    def aug_test(self, imgs, img_metas, rescale=False):
```

Comment:
Since this is a general API, the function should be moved to `SingleStageDetector` thus RepPoints and RetinaNet could reduce duplicated code.

Reply:
The code is duplicate but simple to raise NotImplementedError from unsupported detectors.
I would like to move the function after supporting more detectors on another PR.